src/main/java/edu/kis/powp/jobs2d/drivers/adapter/BaseAdapter.java

```diff
@@ -0,0 +1,57 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.Objects;
+
+/**
+ * driver adapter to drawer
+ */
+public class BaseAdapter implements Job2dDriver {
```

Comment:
The name does not indicate the purpose of the adapter or what was actually adapted.